hw/top_earlgrey/ip/ast/data/ast.hjson

```diff
@@ -52,7 +52,7 @@
       swaccess: "ro",
       hwaccess: "hro",
       tags: [ // don't write random data to any of the AST registers
-              "excl:CsrAllTests:CsrExclAll" ],
+        "excl:CsrAllTests:CsrExclAll" ],
```

Comment:
Oh! It looks like Marno actually did this back in September anyway :-) I'm force-pushing to your branch to drop this commit (since the work is done already).